docs/gsoc.rst

```diff
@@ -184,6 +184,15 @@ which provide outlines on the larger tasks that we're hoping to accomplish.
 .. _these issues: https://github.com/readthedocs/readthedocs.org/issues?direction=desc&labels=Feature&page=1&sort=updated&state=open
 .. _milestones: https://github.com/readthedocs/readthedocs.org/milestones
 
+Some previous year project selected in Google Summer Of Code  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think this should be a separate sub-section. So It should be `----------` instead of `~~~~~~`.
You can read more about this here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections